src/js/component/footer/styles.js

```diff
@@ -4,15 +4,15 @@ const styles = (theme) => {
     footerWrapper: {
       background: secondaryColor?.footerBgColor,
       color: secondaryColor?.contrastText,
+      padding: '16px',
```

Comment:
Let's create material theme at root and try to use as maximum styles and colors as possible